redisvl/index/index.py

```diff
@@ -814,8 +837,21 @@ def batch_query(
             all_parsed.append(parsed)
         return all_parsed
 
+    def _validate_query(self, query: BaseQuery) -> None:
```

Comment:
Could probably be moved to the base search index class since it's shared?

Reply:
Agreed!